ui/lib/core/addon/templates/components/box-radio.hbs

```diff
@@ -1,64 +1,66 @@
 {{#if @disabled}}
-<div class="box-radio-spacing">
   <ToolTip
     @verticalPosition="above"
     @horizontalPosition="center"
     as |T|>
     <T.trigger @tabindex="-1">
-      <label
-        for={{@type}}
-        class="box-radio is-disabled is-marginless"
-        data-test-mount-type-radio
-        data-test-mount-type={{@type}}
+      <div
+        class="box-radio is-disabled"
+        data-test-mount-type={{@key}}
+        tabindex="0"
       >
         <Icon
           @glyph={{@glyph}}
           @size="xl"
           class="has-text-grey-light"
         />
-        {{@displayName}}
-        <RadioButton
```

Comment:
Is this the only place `RadioButton` was still in use? I think it was provided by an addon, so you may be able to remove the addon if this is the only place.

Reply:
Unfortunately not, there are a few other places that I intend to update in a different PR